### PR TITLE
ft: yellow sticky shape

### PR DESCRIPTION
**Pull request by aimee-annabelle**
Added a yellow sticky shape centered on the page, which will stay on top when scrolled

### CODE REVIEW

**Comment by Ntwalii** [comment]
Please pull dev to get the recent changes

**Comment by josueahadi** [comment]
@aimee-annabelle, there's a conflict in your index.html document that needs to be addressed.

**Review by Ntwalii — APPROVED** [review]
The shape has been implemented successfully.

**Review by gloriaumutoni — APPROVED** [review]

File: src/sections/sticky.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div class="flex justify-center mt-20">
+  <img src="../src/images/sticky-shape.svg" alt="" />
```
Comment (Kingcedru) [inline src/sections/sticky.html]: Try to fill the alt attribute with the image description.
example:
`Yellow shape`

**Review by Kingcedru — APPROVED** [review]

**Review by Kingcedru — CHANGES_REQUESTED** [review]
You can check the review I provided.

**Review by Kingcedru — APPROVED** [review]

**Pull request merged**